packages/web-components/src/stories/tree-node.stories.mdx

```diff
@@ -55,11 +61,18 @@ export const Default = (args) => {
 
 export const Selected = (args) => {
     return html`
+<style>
+rux-tree-node{
```

Comment:
can we remove these? even if the result in sb looks weird without a border? nodes are only supposed to be used inside a tree and I think it's best that show our components in isolation.

also when did we add nodes inside nodes? I thought these just had single nodes.

Reply:
I did that so on the expanded example it actually showed it's arrow and stuff. I can get rid of it on the other ones, but is there any issue with keeping it for the expanded variant? Just so it looks like it would if it's expandable

Reply:
Pushed up a new version. I removed the extra tree-node from the variants (besides expanded) and added a blurb about why the border is missing, and moved tree-node into tree on SB. lmk if you want any of that nixed